internal/ccall/cdt.go

```diff
@@ -125,7 +125,7 @@ func (g *Dtlink) Left() *Dtlink {
 }
 
 func (g *Dtlink) SetLeft(v *Dtlink) {
-	g.c.hl = *(*[8]byte)(unsafe.Pointer(v.c))
+	g.c.hl = *(*[reg_size]byte)(unsafe.Pointer(v.c))
```

Comment:
```go
const wordSize = 4 << (^uintptr(0) >> 63)
```
This idiom allows us to avoid build switches. Also, could you name it `wordSize` ?

Reply:
That's a cool hack! Thanks